test/test.rb

```diff
@@ -196,7 +196,7 @@ def create_corners(p1, p2, p3, p4)
 
     marker_corners.write corners_samples
     cmd_in.write setpoint_samples
-    sleep(0.01)
+    sleep(0.1)
```

Comment:
The sleeps make no sense here. If you're waiting for the new sample, the assert_has_one_new_sample is already waiting. Maybe need to set the assert's timeout (last argument) higher ?